submissions/KostiantynOkhorzin/memory-game/script.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const gameBody = document.querySelector('.game__body');
+let playerSteps = 0;
+const cardProcessingTime = 1500;
+
+const dataImgs = [
```

Comment:
You store each item twice here. It doesn't make any sense and should be done in JS.

Reply:
You should only store unique items here and double them in your code.